core/boot.go

```diff
@@ -211,7 +225,12 @@ func ConnectDB() (*sqlx.DB, error) {
 // LoadConf plugins.json into a usable struct.
 func LoadConf() (*PluginJSON, error) {
 	ipath := "github.com/itsabot/abot"
-	p := filepath.Join(os.Getenv("GOPATH"), "src", ipath, "plugins.json")
+	path := os.Getenv("GOPATH")
+	if path == "" {
```

Comment:
After we've confirmed that GOPATH is set in LoadEnvVars, we shouldn't need subsequent `if` checks in other functions.